qiskit/transpiler/target.py

```diff
@@ -89,7 +92,7 @@ def calibration(self):
     def calibration(self, calibration: Union[Schedule, ScheduleBlock, CalibrationEntry]):
         if isinstance(calibration, (Schedule, ScheduleBlock)):
             new_entry = ScheduleDef()
-            new_entry.define(calibration)
+            new_entry.define(calibration, user_provided=True)
```

Comment:
Yes. If a provider defines a backend schedule, one can directly create `CalibrationEntry` instance with `user_provided=False`. This code block is invoked only when `.calibration` setter is called with raw `Schedule` or `ScheduleBlock`, which is likely the workflow of the end users.

Reply:
Then can we document this behavior? I think it's a bit non-obvious without reading the code. I agree the typical backend construction should be populating the calibration via the `InstructionProperties` constructor but we should make it clear that if a provider were to add calibrations after initial construction via the setter it will be treated as a custom calibration.

Reply:
Added in [f203f59](https://github.com/Qiskit/qiskit-terra/pull/9587/commits/f203f59399989bc7998a94269f837dc2149ad763). I think setter docs is not rendered in Sphinx build, so added detailed note section to the property method.